src/Stratis.Bitcoin.Features.RPC/Controllers/FullNodeController.cs

```diff
@@ -205,36 +205,36 @@ public BlockHeaderModel GetBlockHeader(string hash, bool isJsonFormat = true)
         /// Returns information about a bitcoin address
         /// </summary>
         /// <param name="address">bech32 or base58 BitcoinAddress to validate.</param>
-        /// <returns>JObject containing a boolean indicating address validity</returns>
+        /// <returns>ValidatedAddress containing a boolean indicating address validity</returns>
         [ActionName("validateaddress")]
         [ActionDescription("Returns information about a bech32 or base58 bitcoin address")]
-        public JObject ValidateAddress(string address)
+        public ValidatedAddress ValidateAddress(string address)
         {
             if (string.IsNullOrEmpty(address))
                 throw new ArgumentNullException("address");
 
-            var res = new JObject();
-            res["isvalid"] = false;
+            var res = new ValidatedAddress();
+            res.IsValid = false;
```

Comment:
This is implied if the bool is not specifically set to false ;) so you can remove this line 👍